src/accessories/CampaignImporterExporter.ttslua

```diff
@@ -0,0 +1,291 @@
+local campaignTokenData = {
+  GUID = "51b1c9",
+  Name = "Custom_Model",
+  Transform = {
+    posX = -21.25,
+    posY = 1.68,
+    posZ = 55.59,
+    rotX = 0,
+    rotY = 270,
+    rotZ = 0,
+    scaleX = 2,
+    scaleY = 2,
+    scaleZ = 2
+  },
+  Nickname = "Arkham Coin",
+  Description = "SCED Importer Token",
+  GMNotes = "",
+  Tags = {
+    "ImporterToken"
+  },
+  CustomMesh = {
+    MeshURL = "http://cloud-3.steamusercontent.com/ugc/943949966265929204/A38BB5D72419E6298385556D931877C0A1A55C17/",
+    DiffuseURL = "http://cloud-3.steamusercontent.com/ugc/254843371583188147/920981125E37B5CEB6C400E3FD353A2C428DA969/",
+    NormalURL = "",
+    ColliderURL = "http://cloud-3.steamusercontent.com/ugc/943949966265929204/A38BB5D72419E6298385556D931877C0A1A55C17/",
+    Convex = true,
+    MaterialIndex = 2,
+    TypeIndex = 0,
+    CustomShader = {
+    SpecularColor = {
+      r = 0.7222887,
+      g = 0.507659256,
+      b = 0.339915335
+    },
+    SpecularIntensity = 0.4,
+    SpecularSharpness = 7.0,
+    FresnelStrength = 0.0
+    },
+    CastShadows = true
+  }
+}
+
+-- counter GUIDS (4x damage and 4x horror)
+local DAMAGE_HORROR_GUIDS = {
+  "eb08d6"; "e64eec"; "1f5a0a"; "591a45";
+  "468e88"; "0257d9"; "7b5729"; "beb964";
+  }
+
+local chaosBagApi
+local playAreaAPI
+local optionPanelApi
+local deckImporterApi
+
+local campaignBoxGUID
+
+function onLoad(save_state)
+  chaosBagApi = require("chaosbag/ChaosBagApi")
+  playAreaAPI = require("core/PlayAreaApi")
+  deckImporterApi = require("arkhamdb/DeckImporterApi")
+  optionPanelApi = require("core/OptionPanelApi")
+  blessCurseApi = require("chaosbag/BlessCurseManagerApi")
```

Comment:
I'm not sure if this actually makes a difference functionally or performance-wise, but let's move these out of `onLoad()` (e.g. directly to line 49-52), since that's the syntax we've used everywhere else.